### ISSUE TITLE
schema[playbook] on vars_files

### ISSUE THREAD

**Issue by diomedesadmin** [ansible/ansible-lint#4328]
```
  vars_files:
    - "../../secrets/secrets-prod.yml"
```

```
schema[playbook]: $[0].vars_files ['../../secrets/secrets-prod.yml'] is not of type 'object'
playbooks/prod/clockwork_reference_clock.base.yml:1  Returned errors will not include exact line numbers, but they will mention
the schema name being used as a tag, like ``schema[playbook]``,
``schema[tasks]``.

This rule is not skippable and stops further processing of the file.

If incorrect schema was picked, you might want to either:

* move the file to standard location, so its file is detected correctly.
* use ``kinds:`` option in linter config to help it pick correct file type.


Read documentation for instructions on how to ignore specific rule violations.

               Rule Violation Summary
 count tag              profile rule associated tags
     1 schema[playbook] basic   core

Failed: 1 failure(s), 


```

Tried everything, without '../../', single string, full path. Had to add `schema[playbook]` into the skip_list.

**Comment by alisonlhart** [comment]
Closing this as we are unable to reproduce. Please check the schema file [here](https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/schemas/ansible.json#L641) to ensure you're using valid types.

**Comment by vv0s4k** [comment]
@alisonlhart I ran into probably same issue today. Here is how you can reproduce it: 
```yaml
---
- name: Create database and user
  hosts: all
  become: false
  gather_facts: true
  gather_subset: min
  vars_files:
    - ../../roles/services_deployment/defaults/main.yml
```
Obviously `gather_subset: min` is not correct here but the `$[0].vars_files ['../../roles/services_deployment/defaults/main.yml'] is not of type 'object'` error was so much confusing that it took me quite some time to realize I made a mistake earlier and there is nothing wrong with vars_files. 
Once I changed `gather_subset: min` to 
```yaml
  gather_subset:
    - min
```
the linter stopped complaining.
